to.holepunch.Keet.yml

```diff
@@ -0,0 +1,52 @@
+id: to.holepunch.Keet
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: /app/bin/usr/bin/keet
```

Comment:
comman should be in `/app/bin` and here it should be a simple command name.

Reply:
The keet binary requires a certain directory structure to work correctly (the one from /assets). The path to the assets is hardcoded [in the C code](https://github.com/holepunchto/libpear/blob/c9931ebf26aa48a171276edfab983aa4d745542b/src/pear.c#L105). I guess we could change the name of the directories to avoid the redundant "bin" in the flatpak? How would you prefer it to be called?

Reply:
this is unreliable in general. `/usr` is a prefix and finding it anywhere down is not standard. Here the prefix is `/app`.

Maybe you should be from source if that coded like that. at least the flatpak content is immutable.

Reply:
sorry I don't understand what you wrote, could you please rephrase that?

Reply:
> assets is hardcoded [in the C code](https://github.com/holepunchto/libpear/blob/c9931ebf26aa48a171276edfab983aa4d745542b/src/pear.c#L105).

This should be fixed

Reply:
I'm not sure I understand, do you want the assets used by the app upstream to be stored in flatpak-specific paths?

Reply:
$prefix is not Flatpak specific.